In [2]:
import requests

response = requests.get("https://api.airvisual.com/v2/cities?state={{STATE_NAME}}&country={{COUNTRY_NAME}}&key={{bd58b427-3eee-40a6-aa84-27b9949e09f1}}")
# Print the status code of the response.
print(response.status_code)

403


* 200 — everything went okay, and the result has been returned (if any)
* 301 — the server is redirecting you to a different endpoint. This can happen when a company switches domain names, or an endpoint name is changed.
* 401 — the server thinks you’re not authenticated. This happens when you don’t send the right credentials to access an API (we’ll talk about authentication in a later post).
* 400 — the server thinks you made a bad request. This can happen when you don’t send along the right data, among other things.
* 403 — the resource you’re trying to access is forbidden — you don’t have the right permissions to see it.
* 404 — the resource you tried to access wasn’t found on the server.

In [3]:
response = requests.get("https://api.airvisual.com/v2/states?country={{COUNTRY_NAME}}&key={{bd58b427-3eee-40a6-aa84-27b9949e09f1}}")
# Print the status code of the response.
print(response.status_code)

403
